sql.txt

```diff
@@ -0,0 +1,99 @@
+BÀI TẬP SQL
+1. Viết lệnh sql để tạo dữ liệu cho các bảng
+	- bảng project_users 
+	CREATE TABLE project_users (
+		id bigint NOT NULL,
+		project_id bigint,
+		user_id bigint,
+		creat_at timestamp,
```

Comment:
sai tên trường create_at => created_at
cần add thêm NOT NULL DEFAULT CURRENT_TIMESTAMP